docs/development_guide/targets.rst

```diff
@@ -3,7 +3,7 @@ Targets
 
 To facilitate encapsulation and reuse of schema parameters related to design targets, SiliconCompiler implements a :meth:`.load_target()` function which can run scripts that set up common combinations of :class:`.Flow`, :class:`.PDK`, :class:`.Library`, and :class:`.Checklist` modules.
 
-The :meth:`.load_target()` function takes in a string ``targetname``. Similar to most other SiliconCompiler modules, it will search for a ``targets.targetname`` module in the current directory and Python search path. A full list of built-in targets can be found on the :ref:`Targets directory` page.
+The :meth:`.load_target()` function takes in a string ``targetname``. Similar to most other SiliconCompiler modules, it will search for a ``targets.targetname`` module in the current directory and Python search path. A full list of built-in targets can be found on the :ref:`targets_directory` page.
```

Comment:
Should update this since I think the `load_target()` mechanism has changed a bit now (can either provide a module directly, or string that's a full import path or something under `siliconcompiler.targets.<target>`)

Reply:
Awesome! I'll make a note to get this fixed in the next docs update!